### ISSUE TITLE
Error while building on an ec2 in elastic beanstalk

### ISSUE THREAD

**Issue by jredbeard** [steemit/leeroy#2]
The error `ApplyLayer exit status 1 stdout:  stderr: unexpected EOF` is dropped after attempting a build. I believe this is an error coming from docker.

**Comment by jnordberg** [comment]
Was the docker socket forwarded from the host with `-v /var/run/docker.sock:/var/run/docker.sock`?

**Comment by jredbeard** [comment]
Yes - and I can confirm it was actually building... After setting logging to 'debug' you could see the build steps. It would complete and then drop this error message at the end.

**Comment by jnordberg** [comment]
Odd, might have been some problem connecting the registry during/after build then. Did you check if a `docker build` on the host machine worked?
